pkg/commands/handler.go

```diff
@@ -244,3 +258,70 @@ func getConnection(ctx context.Context) (*pgx.Conn, func() error, error) {
 
 	return pg, close, nil
 }
+
+func handleFailover(w http.ResponseWriter, r *http.Request) {
+	res := &Response{Result: true}
+
+	render.JSON(w, res, http.StatusOK)
+}
+
+func handleRestart(w http.ResponseWriter, r *http.Request) {
+
+	args := []string{"stolon", "pg_ctl", "-D", "/data/postgres", "restart"}
+
+	cmd := exec.Command("gosu", args...)
+
+	if err := cmd.Run(); err != nil {
+		render.Err(w, err)
+		return
+	}
+
+	if cmd.ProcessState.ExitCode() != 0 {
+		err := fmt.Errorf(cmd.ProcessState.String())
+		render.Err(w, err)
+		return
+	}
+
+	res := &Response{Result: "Restart completed successfully"}
+
+	render.JSON(w, res, http.StatusOK)
+}
+
+func handleSettings(w http.ResponseWriter, r *http.Request) {
+	res := &Response{Result: true}
+
+	render.JSON(w, res, http.StatusOK)
+}
+
+func handleRole(w http.ResponseWriter, r *http.Request) {
```

Comment:
What are your thoughts on converting this into an admin function?  https://github.com/fly-apps/postgres-ha/blob/7bea596565bec916349c682e5b4025ec11f630e8/pkg/flycheck/role.go#L35-L47

Thinking it could be nice to avoid the second http call.

Reply:
Yeess